Task70/YandexLoginLogout/YandexLoginLogout/ConfigData.cs

```diff
@@ -9,6 +9,7 @@ public class ConfigData
     {        
         public readonly int WaitTime = 3;
         public readonly Dictionary<string, User> Users;
+        public readonly string ScreenshotPath = "D:\\SeleniumScreenshots\\";
```

Comment:
Please, don't use local paths anymore within your project. It will cause issues if someone else download your project it try to run it. Instead of that I would recommend you to use relative project [path](https://stackoverflow.com/questions/816566/how-do-you-get-the-current-project-directory-from-c-sharp-code-when-creating-a-c).

Reply:
Thanks for the link! I thought about relative path but I realy didn't know how to implement it. I saw that SaveAsFile method understands only full path and doesn't perceive the syntax like ../../